es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
 <template>
     <div>
-        atoms - layout
+        <h1>
+            Layout
+        </h1>
+        <p>
+            Extended from 
+            <NuxtLink to="https://bootstrap-vue.org/docs/components/layout" target="_blank">
```

Comment:
Nit: we should set up automatic formatting for this but can we use the Vue snake-case convention of `<nuxt-link>` for this instead?